kahuna/public/js/edits/service.js

```diff
@@ -105,8 +105,12 @@ service.factory('editsService',
 
         return resource.put({ data }).then(edit =>
             getSynced(originalImage, newImage => matches(edit, newImage))).
-            then(edit => {
+            then(({ edit, image }) => {
+
                 runWatcher(resource, 'update-end');
+
+                $rootScope.$emit('image-updated', image, originalImage);
```

Comment:
This is a bit of an oddity of us having - now - two ways of emitting events.
@kenoir and I think it would be worth taking a breath and trying to untangle some of this mess as it's not only creating multiple paths to the same ends, there are couplings that are making things hard to move.

I personally would like to move this logic to the image service, which would expose a metadata object (similar to the [`usageRights`](https://github.com/guardian/grid/blob/master/kahuna/public/js/image/service.js#L10)) which would work in a silo with the UI . The image service would use the [edits service events](https://github.com/guardian/grid/pull/922/files#diff-81f57b8057236574f9266cb5d4cb54e3R23) to know it's been updated and update itself.